server/seed_reviews.js

```diff
@@ -2,7 +2,7 @@ const faker = require('faker');
 const { Review } = require('./db.js');
 
 const reviewText = () => {
-  let reviewParagraphs;
+  let reviewParagraphs = '';
   const reviewLength = Math.ceil(Math.random() * 4);
```

Comment:
May be more reliable to use the .random from the lodash library?

Reply:
How so?